index.bs

```diff
@@ -5750,6 +5765,104 @@ The [=remote end steps=] with |command parameters| are:
 
 </div>
 
+#### The emulation.setLocaleOverride Command ####  {#command-emulation-setLocaleOverride}
+
+The <dfn export for=commands>emulation.setLocaleOverride</dfn> command modifies
+locale on the given top-level traversables or user contexts.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      emulation.SetLocaleOverride = (
+        method: "emulation.setLocaleOverride",
+        params: emulation.SetLocaleOverrideParameters
+      )
+
+      emulation.SetLocaleOverrideParameters = {
+        locale: text / null,
+        ? contexts: [+browsingContext.BrowsingContext],
+        ? userContexts: [+browser.UserContext],
+      }
+    </pre>
+   </dd>
+   <dt>Result Type</dt>
+   <dd>
+    <code>
+     EmptyResult
+    </code>
+   </dd>
+</dl>
+
+<div>
+To <dfn>set emulated locale</dfn> given |navigable| and |locale|:
+
+1. If |locale| is null, [=map/remove=] |navigable| from
```

Comment:
For other emulation commands we do not have a global map from the navigable to the feature. Why do we need it here?